src/natcap/invest/__init__.py

```diff
@@ -30,176 +46,188 @@ class _MODELMETA:
         gui='routedem.RouteDEM',
         userguide='routedem.html',
         aliases=()),
     'scenario_generator_proximity': _MODELMETA(
-        model_title='Scenario Generator: Proximity Based',
+        model_title=_('Scenario Generator: Proximity Based'),
         pyname='natcap.invest.scenario_gen_proximity',
         gui='scenario_gen.ScenarioGenProximity',
         userguide='scenario_gen_proximity.html',
         aliases=('sgp',)),
     'scenic_quality': _MODELMETA(
-        model_title='Unobstructed Views: Scenic Quality Provision',
+        model_title=_('Unobstructed Views: Scenic Quality Provision'),
         pyname='natcap.invest.scenic_quality.scenic_quality',
         gui='scenic_quality.ScenicQuality',
         userguide='scenic_quality.html',
         aliases=('sq',)),
     'sdr': _MODELMETA(
-        model_title='Sediment Delivery Ratio',
+        model_title=_('Sediment Delivery Ratio'),
         pyname='natcap.invest.sdr.sdr',
         gui='sdr.SDR',
         userguide='sdr.html',
         aliases=()),
     'seasonal_water_yield': _MODELMETA(
-        model_title='Seasonal Water Yield',
+        model_title=_('Seasonal Water Yield'),
         pyname='natcap.invest.seasonal_water_yield.seasonal_water_yield',
         gui='seasonal_water_yield.SeasonalWaterYield',
         userguide='seasonal_water_yield.html',
         aliases=('swy',)),
     'stormwater': _MODELMETA(
-        model_title='Stormwater',
+        model_title=_('Stormwater'),
         pyname='natcap.invest.stormwater',
         gui='stormwater.Stormwater',
         userguide='stormwater.html',
         aliases=()),
     'wave_energy': _MODELMETA(
-        model_title='Wave Energy Production',
+        model_title=_('Wave Energy Production'),
         pyname='natcap.invest.wave_energy',
         gui='wave_energy.WaveEnergy',
         userguide='wave_energy.html',
         aliases=()),
     'wind_energy': _MODELMETA(
-        model_title='Wind Energy Production',
+        model_title=_('Wind Energy Production'),
         pyname='natcap.invest.wind_energy',
         gui='wind_energy.WindEnergy',
         userguide='wind_energy.html',
         aliases=()),
     'urban_flood_risk_mitigation': _MODELMETA(
-        model_title='Urban Flood Risk Mitigation',
+        model_title=_('Urban Flood Risk Mitigation'),
         pyname='natcap.invest.urban_flood_risk_mitigation',
         gui='urban_flood_risk_mitigation.UrbanFloodRiskMitigation',
         userguide='urban_flood_risk_mitigation.html',
         aliases=('ufrm',)),
     'urban_cooling_model': _MODELMETA(
-        model_title='Urban Cooling',
+        model_title=_('Urban Cooling'),
         pyname='natcap.invest.urban_cooling_model',
         gui='urban_cooling_model.UrbanCoolingModel',
         userguide='urban_cooling_model.html',
         aliases=('ucm',)),
 }
 
 
+def install_language(language_code):
+    """Globally install the _() function for the requested language."""
```

Comment:
Good idea, I added a `ValueError` if the language code isn't supported. You're right, the fallback applies to each string individually. That's helpful so that we can use an incomplete translation set if necessary.